source/scripts/Utility.js

```diff
@@ -6,3 +6,20 @@ module.exports.queryURL = function(name) {
     var results = new RegExp("[\\?&]" + name + "=([^&#]*)").exec(location.search)
     return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "))
 }
+
+
+module.exports.Point = class Point {
+    constructor(point = {}) {
+        this.x = point.x || 0
+        this.y = point.y || 0
+    }
+    toString() {
+        return this.x + "x" + this.y
+    }
+    toPoint(that = {}) {
```

Comment:
Would `translate` have the connotation of mutating the point? I want a method that returns a separate point, that has been translated.

Also, as a note, we're not even using this Point class anywhere yet. :x